src/maps/dateRangeSelectValue.ts

```diff
@@ -0,0 +1,44 @@
+import { DateRangeSelectRangeValue, DateRangeSelectSpanValue, DateRangeSelectValue } from '@prefecthq/prefect-design'
+import { differenceInSeconds, addSeconds } from 'date-fns'
+import { MapFunction } from '@/services/Mapper'
+import { sortDates } from '@/utilities/dates'
+
+function nowWithoutMilliseconds(): Date {
+  const now = new Date()
+
+  now.setMilliseconds(0)
+
+  return now
+}
+
+type DateRange = {
+  startDate: Date,
+  endDate: Date,
+  timeSpanInSeconds: number,
+}
+
+function mapDateRangeSelectRangeValue({ startDate, endDate }: DateRangeSelectRangeValue): DateRange {
+  const timeSpanInSeconds = differenceInSeconds(endDate, startDate)
+
+  return { startDate, endDate, timeSpanInSeconds }
+}
+
+function mapDateRangeSelectSpanValue({ seconds }: DateRangeSelectSpanValue): DateRange {
```

Comment:
Can we change the name of these to match the format other mappers are using?
`mapDateRangeSelectRangeValue` => `mapDateRangeSelectRangeValueToDateRange`
and
`mapDateRangeSelectSpanValue` => `mapDateRangeSelectSpanValueToDateRange`
? 

Also might not hurt to make these available to the global mapper 🤷🏻

Reply:
Updated the names. Didn't seem like adding them to the global mapper was worth it since there isn't currently a need for mapping these individually. The functions are just organization for the primary mapping function.

Reply:
i thought you might feel that way, nbd imo